src/audio/midi/MidiAudio.cpp

```diff
@@ -109,7 +80,7 @@ try
     // By waiting on the shutdown future with the duration of the note, we'll
     // either be paused for the appropriate amount of time, or we'll break out
     // of the wait early if we've been shutdown.
-    _shutdownFuture.wait_for(duration);
+    _skip.wait(::base::saturated_cast<DWORD>(duration.count()));

```

Comment:
This comment could do with an update. Maybe something like "By waiting on the skip event...", and possibly also mention that it applies to both shutdown and skipping notes.